config/locales/en.yml

```diff
@@ -55,6 +55,10 @@ en:
           attributes:
             pseud_id:
               taken: "is already listed as a creator."
+        user:
+          attributes:
+            password_confirmation:
```

Comment:
Everything on the face of the planet, including simple logic, says this should be `password`. And yet.

Reply:
This is much better, thanks!